build.gradle

```diff
@@ -348,6 +348,7 @@ project(':iceberg-core') {
     implementation libs.jackson.databind
     implementation libs.caffeine
     implementation libs.roaringbitmap
+    implementation libs.jts.core
```

Comment:
Why is this needed?

Reply:
I see, this is used for translation between WKT and WKB.

If we choose to do that translation (we could change from a WKT representation or could not allow defaults for geo fields) then we will need to update license documentation or make support pluggable. Updating documentation involves researching the code that is pulled in by JTS and reviewing the license and notice for every new transitive dependency.

Reply:
We only allow `null` defaults for geospatial types, so we don't need to implement JSON single value parsing for geospatial types. I have removed single value parsing support for geospatial types and also removed JTS.